lib/features/steps/feature_flag_steps.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+# @!group Feature flag steps
+
+# Verifies that the are no feature flags present
+Then('the event has no feature flags') do
+  featureFlags = Maze::Helper.read_key_path(Maze::Server.errors.current[:body], "events.0.featureFlags")
```

Comment:
Variable naming would be more consistent as `feature_flags` and not violate Rubocop, if we turned it on.